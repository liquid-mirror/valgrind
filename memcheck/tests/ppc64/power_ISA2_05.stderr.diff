--- power_ISA2_05.stderr.exp	2012-09-07 05:51:27.000000000 -0700
+++ power_ISA2_05.stderr.out	2012-11-11 08:05:54.812599000 -0800
@@ -1,10 +1 @@
-
-
-HEAP SUMMARY:
-    in use at exit: 0 bytes in 0 blocks
-  total heap usage: 0 allocs, 0 frees, 0 bytes allocated
-
-For a detailed leak analysis, rerun with: --leak-check=full
-
-For counts of detected and suppressed errors, rerun with: -v
-ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
+valgrind: foo: command not found
