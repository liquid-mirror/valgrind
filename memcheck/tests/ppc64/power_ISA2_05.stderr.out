valgrind: foo: command not found
